src/components/tipRecords/TipRecord.vue

```diff
@@ -187,27 +221,51 @@ export default {
 .comment__icon {
   margin-right: 0.2rem;
   vertical-align: top;
+  height: 1rem;
 }
 
-.tip__footer {
+.tip-footer {
   border-bottom-left-radius: 0.25rem;
   border-bottom-right-radius: 0.25rem;
   color: $light_font_color;
-  font-size: 0.8rem;
-  padding: 1.4rem 1rem 0.75rem;
-}
-
-.tip__footer_wrapper {
+  font-size: 1rem;
+  padding: 0.75rem 1rem 0.75rem;
   display: flex;
   flex-wrap: wrap;
-  justify-content: center;
+  cursor: default;
+
+  ::v-deep .tip-input .button img {
+    height: 0.9rem;
+  }
+
+  .actions-wrapper {
+    display: flex;
+    justify-content: space-evenly;
+    flex-grow: 1;
+    padding: 0 2.3rem;
+
+    .actions {
```

Comment:
```suggestion
    .action {
```
or even `.button-plain`